RELEASE.md

```diff
@@ -31,7 +31,12 @@ the process was done to ensure that we have an aligned process under the org (si
 1. Create a new tag with the correct version from the new `release-<MAJOR.MINOR>` branch
 2. Add the changelog on it and publish. Now, the code source is released !
 
-### Announce the new release:
+### Update the website docs (https://book.kubebuilder.io/quick-start.html)
+
+1. Push a PR to update the `book-v3` branch with the changes of the latest release branch created (`release-<MAJOR.MINOR>`)
+2. Ping in the channel and ask for reviews.
```

Comment:
Do we already reference "the channel" earlier in the document? It might be nice to do something like:
```suggestion
2. Ping in the [Kubebuilder Slack channel](https://kubernetes.slack.com/archives/CAR30FCJZ) and ask for reviews.
```